client/src/game/Game.js

```diff
@@ -220,7 +220,23 @@ class Game extends Phaser.Scene {
         });
 
         damageParticles.setDepth(this.renderOrder.particles);
+        const skillUpParticles = this.add.particles("game-atlas");
```

Comment:
consider adding a line break above this, between `damageParticles.setDepth(this.renderOrder.particles);`